### DISCUSSION TITLE
Thanks! (And to show stream-zip documentation)

### DISCUSSION THREAD

**Discussion by michalc** [ministryofjustice/mkdocs-tech-docs-template#11] [Show and tell]
Understood this is still a very prototype-y project, but just to say thanks, it's great so far!

I've used it to document a small package, stream-zip, at the former DIT / current Department for Business and Trade: https://stream-zip.docs.trade.gov.uk/, and I really like it. I know the stream-zip docs probably still have a good amount of rough edges, but still they're much better than they were, thanks in part, or even mostly, to this project.

**Comment by michalc** [comment]
And now added https://stream-unzip.docs.trade.gov.uk/

**Comment by SoumayaMauthoorMOJ** [comment]
Very exciting thanks for letting me know :-)
